youtube-viewer: add page (#4102)
tldr-pages/tldr: pages/common/youtube-viewer.md

# youtube-viewer

> > A command line tool to let you watch youtube.

Reviewer: ```suggestion
> Command-line application for searching and playing videos from YouTube.
```
This is more like the tool's own description, which is usually preferrable.

> More information: <https://github.com/trizen/youtube-viewer>.

- Search for a video:

> `youtube-viewer {{search term}}`

Reviewer: ```suggestion
`youtube-viewer {{search_term}}`
```

> - Login to your youtube account:

Reviewer: ```suggestion
- Login to your YouTube account:
```

`youtube-viewer --login`

> - Watch a video with a specific url in vlc:

Reviewer: ```suggestion
- Watch a video with a specific URL in VLC:
```

`youtube-viewer --player={{vlc}} {{https://youtube.com/watch?v=1dastnond}}`

> - Get to a search prompt and play the video that is selected in 720p:

Reviewer: ```suggestion
- Get to a search prompt and play the selected video in 720p:
```
What do you mean by `get to a search prompt`?

> - Get to a search prompt and play the selected video in 720p:

Reviewer 2: I'm confused by this. What does `Get to a search prompt` mean specifically?

Author: Something like this:
![image](https://user-images.githubusercontent.com/58830309/84422445-05220000-abeb-11ea-8fe6-afc39ca03464.png)

Reviewer 3: @sbrl Does that clarify?

Reviewer 2: Ah, I see! Sorry, this PR slipped under the radar.  It seems like this comment has been resolved further down.

> - Get to a search prompt and play the selected video in 720p:

Reviewer 4: ```suggestion
- Get a search prompt and play the selected video in 720p:
```
Does this sound more nature?

Reviewer: or `Open a search prompt`?

Reviewer 4: `get` or `open` is okay with me.

Reviewer 2: I agree with @zdroid - `Open` sounds better.
```suggestion
- Open a search prompt and play the selected video in 720p:
```

Author: It is a cli search.

Author: Changed to display.

> `youtube-viewer -7`

Reviewer 5: 7 should be in tokens(`{{}}`)

Reviewer: If it's related to 720p, then yes.

Author: Ok added ``-{{7}}``

Author: Yes it is related to 720p.
